daemon/daemon/ct.go

```diff
@@ -35,6 +36,7 @@ func runGC(e *types.Endpoint, prefix string, ctType ctmap.CtType) {
 	fd, err := bpf.ObjGet(file)
 	if err != nil {
 		log.Warningf("Unable to open CT map %s: %s\n", file, err)
+		e.LogStatus(types.Warning, fmt.Sprintf("Unable to open CT map %s: %s", file, err))
```

Comment:
A little difficult, the logger itself has different types of printing the log messages (warning,debug,info, etc). Do you have any ideas on how to do that?